python/pyspark/pandas/generic.py

```diff
@@ -694,7 +694,7 @@ def to_csv(
         escapechar: str, default None
             String of length 1. Character used to escape `sep` and `quotechar`
             when appropriate.
-        num_files: the number of files to be written in `path` directory when
+        repartition: the number of files to be written in `path` directory when
```

Comment:
I think we should just remove this out in the future, and let users to use DataFrame.spark.repartition instead .. instead of adding a new renamed parameter.